Source/Atomic/UI/UI.cpp

```diff
@@ -535,11 +539,52 @@ void UI::HandleUpdate(StringHash eventType, VariantMap& eventData)
         exitRequested_ = false;
         return;
     }
+    
+    mouseStayedTime += eventData[Update::P_TIMESTEP].GetFloat();
+
+    if (mouseStayedTime >= 0.5f)
+    {
+        UIWidget* hoveredWidget = GetHoveredWidget();
```

Comment:
With the TB custom event for hover, we could send the hover widget and capture the tooltip text without wrapping the widget